### PR TITLE
Replace First Look Media references

### LINKED ISSUE #343: Update project maintainers
Freedom of the Press Foundation adopted Dangerzone from Micah Lee (and First Look Media) on mid-2022 ([source](https://freedom.press/news/welcome-to-the-dangerzone/)). As a result, the https://dangerzone.rocks domain got transferred to FPF (https://github.com/freedomofpress/dangerzone.rocks) and some disclaimers were added in documents that the public would read.

Code-wise though, there are still some things left for a full transition to FPF. I'll try to summarize those below, and offer my suggestions. Please correct me if I get something wrong:

### Authors/Maintainers

The author for the Python package (and consequently the rest of the system packages) is:

https://github.com/freedomofpress/dangerzone/blob/9b3d98b20b4ae25cf011fde70b5f60d4c926240a/pyproject.toml#L5

I propose changing this to `["Freedom of the Press Foundation <info@freedom.press>", "Micah Lee <micah.lee@theintercept.com>"]`, since you can refer in this field to the original authors ([source](https://peps.python.org/pep-0621/#authors-maintainers)).

In places where there is only one field for authors/maintainers, e.g.:

https://github.com/freedomofpress/dangerzone/blob/9b3d98b20b4ae25cf011fde70b5f60d4c926240a/setup.py#L22-L23

I propose changing it to:

```
author="Freedom of the Press Foundation",
author_email="info@freedom.press",
```

### Copyright

Currently, the copyright in our license is:

https://github.com/freedomofpress/dangerzone/blob/9b3d98b20b4ae25cf011fde70b5f60d4c926240a/LICENSE#L1-L3

I propose changing this to:

```
MIT License

Copyright (c) 2022-2023 Freedom of the Press Foundation
Copyright (c) 2020-2021 First Look Media
```

### References

1. In our README we offer this URL for more info on Dangerzone: https://tech.firstlook.media/dangerzone-working-with-suspicious-documents-without-getting-hacked. I propose replacing it with https://dangerzone.rocks/about.html, which is more up to date.
2. The publisher homepage is currently https://tech.firstlook.media. I believe the equivalent is https://freedom.press.

### Application URIs

Desktop application typically register themselves under a URI (see [Reverse domain name notation](https://en.wikipedia.org/wiki/Reverse_domain_name_notation)). The URI that Dangerzone uses is currently `media.firstlook.dangerzone`. I propose replacing it with `press.freedom.dangerzone`.

### MacOS Developer ID

The developer ID for MacOS is currently `FIRST LOOK PRODUCTIONS, INC. (P24U45L8P5)`. We should change this to the appropriate one for FPF, most likely `FREEDOM OF THE PRESS FOUNDATION, 501(c)(3)`.

### Windows Developer ID

The project currently refers to a signing certificate that is in possession of First Look Media:

https://github.com/freedomofpress/dangerzone/blob/9b3d98b20b4ae25cf011fde70b5f60d4c926240a/install/windows/build-app.bat#L8-L10

We need to change this to FPF's certificate.

### Windows Registry Keys

The Windows Registry key for Dangerzone is:

https://github.com/freedomofpress/dangerzone/blob/9b3d98b20b4ae25cf011fde70b5f60d4c926240a/install/windows/build-wxs.py#L74-L76

I propose renaming it to `Software\Freedom of the Press Foundation\Dangerzone`. Conversely, we can change the manufacturer from `First Look Media` to `Freedom of the Press Foundation`.

### PR DESCRIPTION
**Pull request by apyrgio**
Update several references to First Look Media in the code, to better reflect the current status, where Freedom of the Press Foundation has taken over the stewardship of the project.

Fixes #343

### CODE REVIEW

**Review by deeplow — COMMENTED** [review]
I tested this on Windows:

It successfully installs over `0.4.0` and I see that the publisher is now FPF, as expected:

Screenshot 2023-02-21 at 12-25-46 Encrypted Code

I see that it leaves some old registry keys but that's just how it is...

image(12)

File: install/macos/build-app.py [hunk 1]
```diff
@@ -74,7 +74,7 @@ def main():
     if args.with_codesign:
         print("○ Code signing app bundle")
         identity_name_application = (
-            "Developer ID Application: FIRST LOOK PRODUCTIONS, INC. (P24U45L8P5)"
+            "Developer ID Application: FREEDOM OF THE PRESS FOUNDATION, 501(c)(3)"
```
[the remark this answers is not in the document]
Comment (apyrgio) [reply install/macos/build-app.py:77]: You're right, I should have added a FIXME there. We're still waiting for the final developer ID. We can either merge the PR once we obtain it, or edit it later. In practice, there won't be much of a difference.

File: install/pyinstaller/pyinstaller.spec [hunk 2]
```diff
@@ -64,7 +64,7 @@ if p == "Darwin":
         coll,
         name="Dangerzone.app",
         icon="../macos/dangerzone.icns",
-        bundle_identifier="media.firstlook.dangerzone",
+        bundle_identifier="press.freedom.dangerzone",
```
[the remark this answers is not in the document]
Comment (apyrgio) [reply install/pyinstaller/pyinstaller.spec:67]: Right. I don't know yet what is the effect of installing Dangerzone 0.4.1 on a system where 0.4.0 is installed (either for Windows or MacOS). I wanted to test it locally, but I haven't managed to do so yet.

[the remark this answers is not in the document]
Comment (deeplow) [reply install/pyinstaller/pyinstaller.spec:67]: I wasn't able to produce `.dmg`s without the signing certificate. And we need that to fully test. Maybe there is a way that I don't know.

**Comment by apyrgio** [comment]
@deeplow: Are you comfortable with merging this PR with some FIXMEs for the signing parts, so that we can test it in our next QA round?

**Comment by deeplow** [comment]
Yes.

File: install/pyinstaller/pyinstaller.spec (hunk 2 above)
[the remark this answers is not in the document]
Comment (apyrgio) [reply install/pyinstaller/pyinstaller.spec:67]: Ok, so we will test this once we have a proper signing certificate. If we find an issue, we can fix it in a separate PR.

**Pull request merged**